systems/robotInterfaces/QPLocomotionPlan.m

```diff
@@ -1,5 +1,5 @@
 classdef QPLocomotionPlan < QPControllerPlanMatlabImplementation
-  properties(GetAccess=private, SetAccess=private)
+  properties(GetAccess=protected, SetAccess=protected)
```

Comment:
private -> protected so we can set them in DRCQPLocomotionPlan. This can be reverted when we have update the LCM decode functions